src/Atmos/Model/AtmosModel.jl

```diff
@@ -260,23 +333,23 @@ include("linear.jl")
 include("remainder.jl")
 include("courant.jl")
 
-"""
-    flux_nondiffusive!(m::AtmosModel, flux::Grad, state::Vars, aux::Vars,
-                       t::Real)
+@doc """
+    flux_nondiffusive!(
+        m::AtmosModel, 
+        flux::Grad, 
+        state::Vars, 
+        aux::Vars,
+        t::Real
+    )
 
 Computes flux non-diffusive flux portion of `F` in:
 
 ```
 ∂Y
--- = - ∇ • (F_{adv} + F_{press} + F_{nondiff} + F_{diff}) + S(Y)
+-- = - ∇ • (F_{nondiff} + F_{diff} + F_{rad} + F_{precip}) + S(Y)
```

Comment:
I'm starting to think that this should just be a simpler doc string. The "how to build a balance law" can include some of these finer details.

Reply:
Maybe just write this similar to the `flux_diffusive!`

Reply:
I'll remove this in favour of the ```math``` equation sets.